src/Common/Admin/Conditional_Content/Black_Friday.php

```diff
@@ -0,0 +1,287 @@
+		 * @param Black_Friday $this The Black Friday instance.
+		 */
+		do_action( 'tec_conditional_content_black_friday', 'wide_banner', $this );
+
+		echo $this->get_wide_banner_html(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped,StellarWP.XSS.EscapeOutput.OutputNotEscaped
+	}
+
+	/**
+	 * Gets the content for the Black Friday promo.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	protected function get_narrow_banner_html(): string {
+		$template_args = [
+			'image_src' => tribe_resource_url( 'images/hero-section-narrow.jpg', false, null, \Tribe__Main::instance() ),
+			'link'      => 'https://evnt.is/tec-bf-2024',
+			'is_narrow' => true,
+			'nonce'     => $this->get_nonce(),
+			'slug'      => $this->slug,
+		];
+
+		return $this->get_template()->template( 'black-friday', $template_args, false );
+	}
+
+	/**
+	 * Render the narrow banner HTML.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function render_narrow_banner_html(): void {
+		if ( ! $this->should_display() ) {
+			return;
+		}
+		/**
+		 * Fires before the narrow banner is rendered.
+		 * This hook is used to add additional content before the narrow banner.
+		 *
+		 * @since TBD
+		 *
+		 * @param string       $slug The slug of the conditional content.
+		 * @param Black_Friday $this The Black Friday instance.
+		 */
+		do_action( 'tec_conditional_content_black_friday', 'narrow_banner', $this );
+
+		echo $this->get_narrow_banner_html(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped,StellarWP.XSS.EscapeOutput.OutputNotEscaped
+	}
+
+	/**
+	 * Include the Black Friday promo in the tickets settings section.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	public function include_tickets_settings_section(): void {
+		if ( ! $this->should_display() ) {
+			return;
+		}
+
+		$page = tribe_get_request_var( 'page' );
+		if ( $page !== 'tec-tickets-settings' ) {
+			return;
+		}
+
+		$this->render_narrow_banner_html();
+	}
+
+	/**
+	 * Replace the opening markup for the general settings info box.
+	 *
+	 * @since TBD
+	 *
+	 * @param Settings_Sidebar $sidebar Sidebar instance.
+	 *
+	 * @return void
+	 */
+	public function include_sidebar_section( $sidebar ): void {
+		// Check if the content should currently be displayed.
+		if ( ! $this->should_display() ) {
+			return;
+		}
+
+		$year = date_i18n( 'Y' );
+
+		/**
+		 * Fires before the settings sidebar is rendered.
+		 * This hook is used to add additional content before the narrow banner.
+		 *
+		 * @since TBD
+		 *
+		 * @param string       $slug The slug of the conditional content.
+		 * @param Black_Friday $this The Black Friday instance.
+		 */
+		do_action( 'tec_conditional_content_black_friday', 'settings-sidebar', $this );
+
+		/* translators: %1$s: Black Friday year */
+		$translated_title = sprintf(
+			esc_attr_x( '%1$s Black Friday Sale for The Events Calendar plugins, add-ons and bundles.', 'Alt text for the Black Friday Ad', 'tribe-common' ),
```

Comment:
This should fix the phpcs issue:
```suggestion
		$translated_title = sprintf(
			/* translators: %1$s: Black Friday year */
			esc_attr_x( '%1$s Black Friday Sale for The Events Calendar plugins, add-ons and bundles.', 'Alt text for the Black Friday Ad', 'tribe-common' ),
```